require/requirements_test.go

```diff
@@ -681,3 +683,33 @@ func TestErrorAssertionFunc(t *testing.T) {
 		})
 	}
 }
+
+func TestEventuallyWithTFalse(t *testing.T) {
+	mockT := new(MockT)
+
+	condition := func(collect *assert.CollectT) {
+		True(collect, false)
+	}
+
+	EventuallyWithT(mockT, condition, 100*time.Millisecond, 20*time.Millisecond)
+	if !mockT.Failed {
+		t.Error("Check should fail")
+	}
+}
+
+func TestEventuallyWithTTrue(t *testing.T) {
+	mockT := new(MockT)
+
+	state := 0
+	condition := func(collect *assert.CollectT) {
+		defer func() {
+			state += 1
+		}()
```

Comment:
I think the intent here is to have the delegate called a few times before succeeding. It would be a bit clearer to do the increment outside of a defer to accomplish the same effect
```suggestion
		state += 1
```

Reply:
Done